addons/mrp/models/mrp_production.py

```diff
@@ -337,6 +337,14 @@ def _compute_state(self):
 
         # TODO: duplicated code with stock_picking.py
         for production in self:
+            production_qty = production.product_qty
```

Comment:
qty_produced and product_qty (on production) use the same uom. So i'm not sure the patch should modify compute_state()